servers/collection-api/src/express.ts

```diff
@@ -63,6 +63,13 @@ export async function startServer(port: number): Promise<{
   app.use(
     adminUrl,
     cors<cors.CorsRequest>(),
+    express.json(),
```

Comment:
nit, can you add a small comment what this is doing, something like: "parse incoming JSON body request"